src/components/BalanceBlock/BalanceBlock.component.tsx

```diff
@@ -19,6 +20,7 @@ export const BalanceBlock = ({
   state,
   children,
   typographySx,
+  index,
```

Comment:
What do you think about passing the `img` prop instead of `index` and deciding which icon to show on the `Staking.component.tsx` component level?

Reply:
Ok, but we'll have to import IMG just to pass it as a prop

Reply:
I doesn't sound like a big problem to me.
I think it may be a bit clearer not to have this logic inside a reusable component

Reply:
done